ha/core/system_health/system_health.py

```diff
@@ -440,6 +440,7 @@ def process_event(self, healthevent: HealthEvent):
                 current_health_dict = json.loads(current_health)
                 specific_info = current_health_dict["events"][0]["specific_info"]
                 if (component_type == CLUSTER_ELEMENTS.NODE.value) and specific_info \
+                    and specific_info.get('generation_id', None) \
                     and healthevent.source == HEALTH_EVENT_SOURCES.MONITOR.value:
```

Comment:
This generation_id condition is required to handle first time event process, as now specific_info will contain functional_type only first time  but not generation_id so for 446 line will not found generation id for current_health .